DuggaSys/microservices/sectionedService/getUserDuggaFeedback_ms.php

```diff
@@ -24,6 +24,7 @@
 $courseid=getOP('courseid');
 $moment=getOP('moment');
 $versid = getOP('vers');
```

Comment:
$courseverse is used in the function call (undeclared), but versid is declared up here. I'm assuimg this should be used